src/connectivity.ts

```
@@ -0,0 +1,267 @@
+          // Test model availability
+          text += "### 📋 Model Availability\n\n";
+          try {
+            const availableModels = await provider.listModels();
+            if (availableModels.includes(textModel.modelName)) {
+              text += "> ✅ Selected model is available\n\n";
+            } else {
+              text += "> ⚠️ Selected model not found in available models\n\n";
+              text += "#### Available Models\n\n";
+              availableModels.forEach(model => text += `* \`${model}\`\n`);
+              text += "\n";
+            }
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)